### PR TITLE
Refactor just about everything

### PR DESCRIPTION
**Pull request by oohwooh**
lots of things changed, but it works locally. not perfect by any means but will be way easier to expand on in the future. Note: when merging to your local machine you will have a merge conflict in secrets.py, as there is one of those now.

### CODE REVIEW

**Comment by oohwooh** [comment]
Oh i should also note I forced it to 'utf-8' encoding with the data.json file, as that was the only way to get it to behave on my machine. It shouldn't cause any problems but the code is otherwise functionally identical

File: brains/list/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from .alch import alch
+from .misc import misc
+from .other import other
+from .stock import stock
```
Comment (Yossi) [inline brains/list/__init__.py]: github informs me this is missing a newline at the end of the file

File: commands/craftcmd.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from commands.craft import craft
+
+
```
Comment (Yossi) [inline commands/craftcmd.py]: wouldn't this function be better off living inside craft.py?

File: commands/now.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+from utils import send
+from .decorators import log, send_typing_action
+
+
+@send_typing_action
+@log
+def now(update, context):
+    """what time is it"""
+    text = datetime.utcnow().isoformat()
+    send(text, update, context)
```
Comment (Yossi) [inline commands/now.py]: another missing file ending

File: commands/warehousecmd.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from commands import warehouse
+
+
+def warehousecmd(update, context):
```
Comment (Yossi) [inline commands/warehousecmd.py]: should live in warehouse.py

File: commands/warehousecmd.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+    context.args = update.effective_message.text.split('_')[1:]
+    warehouse(update, context)
```
Comment (Yossi) [inline commands/warehousecmd.py]: end of file missing

File: utils/send.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+            context.bot.send_message(chat_id=chat_id, text=text, parse_mode=ParseMode.HTML, disable_web_page_preview=True)
+    else:
+        kind = filetype.guess(payload.read(261))
+        payload.seek(0)
+        if kind and kind.mime.startswith('image'):
+            logging.info(f'bot said:\n<image>')
+            context.bot.send_photo(chat_id=chat_id, photo=payload)
+        else:
+            logging.info(f'bot said:\n<other>')
+            context.bot.send_document(chat_id=chat_id, document=payload)
```
Comment (Yossi) [inline utils/send.py:22]: eof

File: utils/timewiz/__init__.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+from .game_phase import game_phase
+from .is_witching_hour import is_witching_hour
+from .timewiz import timewiz
```
Comment (Yossi) [inline utils/timewiz/__init__.py]: .

File: secrets.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
+import os
```
Comment (Yossi) [inline secrets.py]: see my ideas on telegram

**Review by Yossi — CHANGES_REQUESTED** [review]
I'm not comfortable with the idea of having a checked in file named secrets.py. It's asking for an accident to happen. Either someone checks in a sensitive secrets.py or a real secrets.py gets clobbered by a pull in the future.

How about instead making a new file that imports from secrets.py if it exists and uses that to populate the default values for os.getenv() and then the code throughout the rest of the bot can import from this new file instead of directly from secrets.py

File: commands/warehousecmd.py (hunk 4 above)
Comment (oohwooh) [reply commands/warehousecmd.py]: I was thinking it makes more sense to keep everything that's a command function in `commands`, but if you would prefer it being in warehouse that's fine

Comment (oohwooh) [reply commands/warehousecmd.py]: oh!!!!!! wait I just realized what you meant. Yeah got it

Comment (oohwooh) [reply commands/warehousecmd.py]: Resolved in 24e2cb9

File: commands/craftcmd.py (hunk 2 above)
Comment (oohwooh) [reply commands/craftcmd.py]: Resolved in 24e2cb9

File: brains/list/__init__.py (hunk 1 above)
Comment (oohwooh) [reply brains/list/__init__.py]: fixed in 7177f07

File: commands/now.py (hunk 3 above)
Comment (oohwooh) [reply commands/now.py]: fixed in 7177f07

File: commands/warehousecmd.py (hunk 5 above)
Comment (oohwooh) [reply commands/warehousecmd.py]: fixed in 7177f07

File: utils/send.py (hunk 6 above)
Comment (oohwooh) [reply utils/send.py:22]: fixed in 7177f07

File: utils/timewiz/__init__.py (hunk 7 above)
Comment (oohwooh) [reply utils/timewiz/__init__.py]: fixed in 7177f07

File: secrets.py (hunk 8 above)
Comment (oohwooh) [reply secrets.py]: made the changes we discussed in 545d4a2 - that look good?

Comment (Yossi) [reply secrets.py]: yep, looks good

**Review by Yossi — APPROVED** [review]
lgtm

**Pull request merged**